generated/nidaqmx/_base_interpreter.py

```diff
@@ -1345,6 +1345,10 @@ def save_scale(self, scale_name, save_as, author, options):
     def save_task(self, task, save_as, author, options):
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def self_cal(self, device_name):
+        raise NotImplementedError
```

Comment:
Please add a test case to catch errors like this. It can be almost identical to the `self_test_device` test case in #463.

Reply:
Added as per #463. Thanks for the pointer